sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/internal/json/json.hpp>
+#include <shared_mutex>
+#include <string>
+
+#include "attestation_client_options.hpp"
+#include <azure/attestation/attestation_client_models.hpp>
```

Comment:
we use `#include < ... >`  for external includes and `#include " ... "`  for same-lib includes.

So, these both includes should use quotes.

The first one, it doesn't matter if the header is in the same path, we would use `#include "azure/attestation/attestation_client_options.hpp"`

Reply:
The rule I've used is "if it should come from `-I` or `INCLUDES`, use <>, otherwise use "". That's the semantics the compiler uses, I believe (it's close)

Reply:
I used to do it in another way as well.  Then @RickWinter made it a guideline https://vhvb1989.github.io/azure-sdk/cpp_implementation.html#cpp-design-physical-include-quotes

And we all follow and honor the guidelines because its something beyond the good, the bad and the ugly  =D